docker/build-base/Dockerfile

```diff
@@ -1,5 +1,4 @@
-ARG PYTHON_VERSION
-FROM python:${PYTHON_VERSION}-slim
+FROM nycplanning/base:latest
```

Comment:
Debated if "build-base" was now redundant and if we could just "merge" it with base, but base takes longer to build (15 min - not that long but a bit annoying if you're waiting on that for your pr tests), and since we don't expect to bump gdal or python version often we can treat it a bit more constant. Meanwhile build-base can still have its python versions bumped and then tested in a dev image in PRs

Reply:
definitely like having a lowest-level image to put things that take a long time to build and that we don't often change in PRs